### PR TITLE
Fixed README and streamlit version

### PR DESCRIPTION
**Pull request by vqmarkman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,13 @@ This repo includes the following components:
 
 ## Project Setup
 
+After you've succesfully cloned OpsCenter reprository, please run:
+
+```
+git submodule init
```
Comment (jacques-n) [inline README.md:23]: I suggest we do a clone that is recursive as opposed to these submodule commands. 

e.g. `git clone --recurse-submodules ...`

File: README.md [hunk 2]
```diff
@@ -17,11 +17,10 @@ This repo includes the following components:
 
 ## Project Setup
 
-After you've succesfully cloned OpsCenter reprository, please run:
+Clone OpsCenter repository by running:
 
 ```
-git submodule init
-git submodule update
+git clone <URL of the repository> --recurse-submodules
```
Comment (jacques-n) [inline README.md:23]: I suggest you include the actual url of the opscenter github repo.

Also not sure but thought flag needed to come before url.

Comment (vqmarkman) [reply README.md:23]: I tested before I posted PR, flag does not have to be before URL.  Will change it though, it reads nicer.

```
vickymarkman@Vickys-MBP-2 Downloads % git clone git@github.com:vqmarkman/OpsCenter.git --recurse-submodules 
Cloning into 'OpsCenter'...
remote: Enumerating objects: 248, done.
```

Do you suggest we write something like this for the URL?
`git@github.com:<git user name>/OpsCenter.git `

Comment (jacques-n) [reply README.md:23]: I'd just use https://github.com/sundeck-io/OpsCenter.git

If someone forks, they'll be able to add their own remote as a second remote.

Comment (vqmarkman) [reply README.md:23]: also found flaky test, looking into it.

**Review by joshelser — APPROVED** [review]
lgtm

**Pull request merged**